Cargo.toml

```
@@ -45,7 +45,7 @@ semver = "~1.0"
 shell-words = "~1.1"
 color-eyre = "~0.6"
 tracing = { version = "~0.1", features = ["attributes", "log"] }
-tracing-subscriber = { version = "~0.3", features = ["env-filter", "time"] }
+tracing-subscriber = { version = "0.3.20", features = ["env-filter", "time"] }
```

Do not pin, the rest is also not pinned.
```suggestion
tracing-subscriber = { version = "~0.3.20", features = ["env-filter", "time"] }
```